simulator/oneshotimporter/importer.go

```diff
@@ -42,17 +58,40 @@ func NewService(e ReplicateService, i ReplicateService) *Service {
 // If you want to use the scheduler configuration along with the imported resources on the simulator,
 // you need to set the path of the scheduler configuration file to `kubeSchedulerConfigPath` value in the Simulator Server Configuration.
 func (s *Service) ImportClusterResources(ctx context.Context, labelSelector metav1.LabelSelector) error {
-	expRes, err := s.exportService.Snap(ctx, labelSelector)
+	for _, gvr := range s.gvrs {
+		if err := s.importResource(ctx, gvr, labelSelector); err != nil {
+			return xerrors.Errorf("import resource %s: %w", gvr.String(), err)
+		}
+	}
+
+	return nil
+}
+
+func (s *Service) importResource(ctx context.Context, gvr schema.GroupVersionResource, labelSelector metav1.LabelSelector) error {
+	selector, err := metav1.LabelSelectorAsSelector(&labelSelector)
 	if err != nil {
-		return xerrors.Errorf("call Snap of the exportService: %w", err)
+		return xerrors.Errorf("convert label selector: %w", err)
 	}
-	impRes, err := snapshot.ConvertResourcesForSnapToResourcesForLoad(expRes)
+
+	resources, err := s.srcDynamicClient.Resource(gvr).List(ctx, metav1.ListOptions{
+		LabelSelector: selector.String(),
+	})
 	if err != nil {
-		return xerrors.Errorf("call ConvertResourcesForSnapToResourcesForLoad: %w", err)
+		return xerrors.Errorf("list resources: %w", err)
 	}
-	// Import to the simulator.
-	if err := s.importService.Load(ctx, impRes, s.importService.IgnoreErr(), s.importService.IgnoreSchedulerConfiguration()); err != nil {
-		return xerrors.Errorf("call Import of the importService: %w", err)
+
+	var wg sync.WaitGroup
+	for _, resource := range resources.Items {
+		wg.Add(1)
+		fmt.Printf("importing resource: %s\n", resource.GetName())
+		go func(r *unstructured.Unstructured) {
+			defer wg.Done()
+			if err := s.resouceApplierService.Create(ctx, r); err != nil {
+				klog.Warningf("failed to import resource: %v", err)
+			}
+		}(&resource)
```

Comment:
`G601: Implicit memory aliasing in for loop. (gosec)` lint error occurs here, but as we use Go 1.22, I guess this error is false-positive.
Worth upgrading golangci-lint?

Reply:
I created the PR: https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/409

Reply:
It's merged and this PR has passed the CI.